system/Session/Handlers/RedisHandler.php

```diff
@@ -75,10 +77,10 @@ public function __construct(AppConfig $config, string $ipAddress)
 
             $this->savePath = [
                 'host'     => $matches[1],
-                'port'     => empty($matches[2]) ? null : $matches[2],
+                'port'     => empty($matches[2]) ? 6379 : $matches[2],
```

Comment:
I recommend a class constant instead of the number.

Reply:
I just set the default port number for Redis, but I think again,
It seems odd that `tcp://127.0.0.1` means `tcp://127.0.0.1:6379`?
Is it better to raise an error?

Reply:
I don't see a problem with that. It looks like it is a common setup for local Redis dev: https://duckduckgo.com/?q=+tcp%3A%2F%2F127.0.0.1%3A6379&t=iphone&ia=qa

Reply:
`tcp://127.0.0.1:6379` is no problem.
But does `tcp://127.0.0.1` equals `tcp://127.0.0.1:6379`?
I doubt it.

Reply:
That, my friend, is beyond me. It looks like the port is pre-specified in the Config file (https://raw.githubusercontent.com/redis/redis/7.0/redis.conf) so my hunch is "yes", but without testing I don't know.

Reply:
I checked the test in this PR with `$sessionSavePath = 'tcp://127.0.0.1'`on PHP 8.0,
and the tests passed.
So to keep compatibility we need to accept `tcp://127.0.0.1` as `tcp://127.0.0.1:6379` in this case.

Reply:
Refactored to use class constant.